docs/ref/csrf.txt

```diff
@@ -79,7 +79,7 @@ The CSRF protection is based on the following things:
    requests from ``www.example.com`` and ``api.example.com``. If the setting is
    not set, then the referer must match the HTTP ``Host`` header.
 
-   Expanding the accepted referers beyond the current host or cookie domain can
+   Expanding the accepted referrers beyond the current host or cookie domain can
```

Comment:
This is not a typo, check out https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer.

Reply:
gotcha and saw in the spelling wordlist referers was there too just wasn't sure in this text file

Reply:
Removed this then